supplementary_style_guide/glossary_terms_conventions/general_conventions/d.adoc

```diff
@@ -612,7 +612,7 @@ Red Hat Directory Server conforms to LDAP standards.
 [discrete]
 [[drl]]
 ==== image:images/yes.png[yes] DRL (noun)
-*Description*: In Red Hat JBoss BRMS and Red Hat JBoss BPM Suite, _DRL_ is an abbreviation for the Drools Rule Language, which is a file with the .drl extension. A DRL file stores technical rules as text and can be managed in the Red Hat Red Hat JBoss BRMS user interface. A DRL file contains one or more rules.
+*Description*: In Red Hat JBoss BRMS and Red Hat JBoss BPM Suite, _DRL_ is an abbreviation for the "Drools Rule Language", which is a file with the .drl extension. A DRL file stores technical rules as text and can be managed in the Red Hat Red Hat JBoss BRMS user interface. A DRL file contains one or more rules.
```

Comment:
This looks odd to me:
"can be managed in the *Red Hat Red Hat* JBoss BRMS user interface". Are there supposed to be two Red Hats in a row like that?

Reply:
Looks to me like someone did a global change from JBoss to Red Hat JBoss without realizing this instance already had a Red Hat?

Reply:
Good catch! I removed the duplicate "Red Hat". 

@sbmetz if you had any comments, I don't see them. Did you still need to hit submit?